en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md

```diff
@@ -104,6 +104,75 @@ Follow the steps below to enable statistics for the **endpoint** artifacts:
 1.  Select the required endpoint artifacts from the project explorer. 
 2.  Select **Statistics Enabled** and (if required) **Trace Enabled** as shown below.
      ![endpoint properties](../../assets/img/ei-analytics/endpoint-properties.png)
+     
+### Configuring the Micro Integrator to Publish Data to Analytics Server
```

Comment:
Is this section only applicable when there is a cluster of EI Nodes?
Because, AFAIR, we didnt need to do this configuration when testing locally?
@ramindu90

Reply:
These are default configs, we need to update it if we have a different one. we can mention it here.